index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>UK News</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
+</head>
+
+<body>
+    <div class="app">
+        <div class="header">
+            <div class="logo">UK News</div>
+            <div class="search_form">
+                <form class="search">
+                    <input id='search' type="text" placeholder="Search all news...">
+                    <button type="submit"><i class="fa fa-search"></i></button>
+                </form>
+            </div>
+        </div>
+        <div class="main">
+            <div id="newsfeed">
+
+            </div>
+            <button onclick="topFunction()" id="myBtn" title="Go to top">Go to Top</button>
```

Comment:
`myBtn` does not tell much about what the button does. Something more descriptive would work better